app/main.py

```diff
@@ -7,13 +7,38 @@
     "The Lord of the Rings: The Return of the King",
     "Pulp Fiction",
     "The Good, the Bad and the Ugly",
-    "The Matrix",
+    "The Matrix"
 ]
 profile_info = ("michel", "michel@gmail.com", "12345678")
 marks = {
     "John": 4,
-    "Sergio": 3,
+    "Sergio": 3
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+all_variables = {}
+all_variables["lucky_number"] = lucky_number
+all_variables["pi"] = pi
+all_variables["one_is_a_prime_number"] = one_is_a_prime_number
+all_variables["name"] = name
+all_variables["my_favourite_films"] = my_favourite_films
+all_variables["profile_info"] = profile_info
+all_variables["marks"] = marks
+all_variables["collection_of_coins"] = collection_of_coins
+
+for values_from_all_variables in all_variables.values():
+    if type(values_from_all_variables) == int \
+            or type(values_from_all_variables) == float \
+            or type(values_from_all_variables) == bool \
+            or type(values_from_all_variables) == str \
+            or type(values_from_all_variables) == tuple:
```

Comment:
I recommend using braces `()` for multiline and avoiding `\`. That doesn't change code logic, but it improves code appearance

Reply:
Can i use this condition? `if isinstance(values_from_all_variables, (int, float, bool, str, tuple)):`

Reply:
indeed, but do you really need conditions for this task?